.github/workflows/ci.yaml

```diff
@@ -87,10 +87,15 @@ jobs:
           - v1.22.17
           - v1.24.17
           - v1.25.16
-          - v1.26.13
+          - v1.26.15
+          - v1.27.16
+          - v1.28.14
+          - v1.29.9
```

Comment:
I tested this change and it seems that the image 1.29.9 is not available 

see test error: https://github.com/DataDog/helm-charts/actions/runs/11447909017/job/31850338128 

1.29.8 is the last 1.29.x available 

```
➜ crane ls kindest/node | grep 1.29` 
v1.29.0
v1.29.1
v1.29.2
v1.29.4
v1.29.7
v1.29.8
```